homeassistant/components/camera/__init__.py

```diff
@@ -736,7 +736,6 @@ async def handle_async_mjpeg_stream(
         return await self.handle_async_still_stream(request, self.frame_interval)
 
     @property
-    @final
```

Comment:
You're not allowed to override the `state` property. The base entity class for each entity integration decides what state is allowed, not the implementing platform.

Reply:
I have reverted this change.

Reply:
This has not been reverted

Reply:
Hi @joostlek,
I have made a commit d9d89b8b6ea38f571763197f605fbda2cc44da01 to revert this line.
It should be part of this PR. Can you please check?